csharp/Facebook.Yoga.Desktop.sln

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
The Universal specific solution is only really useful if you want to run the universal tests... You can build the necessary binaries by using any of the three solutions... I'd be open to opinions on whether having three solutions is 1) a good idea or 2) more hassle than its worth.